### ISSUE TITLE
check_setup.rb incorrect version check

### ISSUE THREAD

**Issue by blucas** [boltops-tools/terraspace#118]
## Checklist

- [x] Upgrade Terraspace: Are you using the latest version of Terraspace? This allows Terraspace to fix issues fast. There's an Upgrading Guide: https://terraspace.cloud/docs/misc/upgrading/
- [x] Reproducibility: Are you reporting a bug others will be able to reproduce and not asking a question. If you're unsure or want to ask a question, do so on https://community.boltops.com
- [ ] Code sample: Have you put together a code sample to reproduce the issue and make it available?  Code samples help speed up fixes dramatically. If it's an easily reproducible issue, then code samples are not needed. If you're unsure, please include a code sample.

## My Environment

| Software         | Version |
| ---------------- | ------- |
| Operating System |     Mac Mojave    |
| Terraform        |    1.0.0     |
| Terraspace       |      0.6.10   |
| Ruby             |     2.7.1    |

---

## Expected Behaviour

When executing a terraspace project using terraform 1.0.0 check_setup succeeds.

## Current Behavior

```
Detected Terrspace version: 0.6.10
Detected Terraform bin: /usr/local/bin/terraform
Detected Terraform v1.0.0
Terraspace requires Terraform v0.12.x and above
The installed version of terraform may not work with terraspace.
Recommend using at least terraform v0.12.x
If you would like to bypass this check. Use TS_VERSION_CHECK=0
```

## Step-by-step reproduction instructions

Setup a new terraspace project which uses terraform 1.0.0, then run a terraspace command such as `bundle exec terraspace all init`

## Code Sample

N/A

## Solution Suggestion

The logic in check_setup.rb must be wrong somewhere.  Unfortunately my ruby skills are lacking so I can't spot what is wrong 😞

**Comment by guilhermepozo** [comment]
Im updated my Terraform to 1.0.0 and i can't use my stacks :fearful:

**Comment by tongueroo** [comment]
Fixed in 0.6.11. Thanks!
